modules/authorization/policy-definitions/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Policy Definitions",
+  "summary": "With this module you can create policy definitions across the management group or subscription scope.",
```

Comment:
Should the README.md be updated to "This module deploys Policy Definitions on a Management Group or Subscription scope." and then this file updated accordingly?